pkg/image/api/types.go

```diff
@@ -306,6 +307,8 @@ type TagEventList struct {
 	Items []TagEvent
 	// Conditions is an array of conditions that apply to the tag event list.
 	Conditions []TagEventCondition
+	// PendingCopy is set if there is a copy waiting for a layer upload on this image stream tag.
+	PendingCopy *ImageStreamTagCopy
```

Comment:
For two step copy (definition then binary layer upload) we need a place to store the pending image metadata.  Since the API is tag focused, it's ok to not allow lots of pending images.  I don't want to put too much metadata in condition